### DISCUSSION TITLE
How to remove the padding when `MinLimit and MaxLimit == null`?

### DISCUSSION THREAD

**Discussion by GF-Huang** [Live-Charts/LiveCharts2#394] [Q&A] [answered]
When the `MinLimit` and `MaxLimit` is `null`, the padding too much.

![image](https://user-images.githubusercontent.com/4510984/153655478-11022df5-9875-4d05-8082-10f35cdb4e53.png)

After I scroll the wheel on it (in other words, `MinLimit` and `MaxLimit` is not `null`), it fill the padding, that looks better.

![image](https://user-images.githubusercontent.com/4510984/153655688-29415418-620a-4c26-aed1-3b48555a82b4.png)

**Comment by beto-rodriguez** [comment, marked as answer]
There is the DataPadding property

https://github.com/beto-rodriguez/LiveCharts2/blob/master/docs/shared/series2.md#datapadding

Set your `Series.DataPadding` property to `new LvcPoint(0, 0)`

**Reply by GF-Huang** [reply]
Is it possible to configurate it global? Instead of configuration at each `Series`. By default, when the `axis.MinLimit` and `axis.MaxLimit` is `null`,  the `DataPadding` is not zero. But when I scroll on the chart, it will become zero. I want it always zero global.

**Reply by mark-monteiro** [reply]
@GF-Huang You can set properties globally using the theme support. For example, you can do something like this when configuring LiveCharts:

```cs
config.AddLightTheme(theme => theme.Style.HasRuleForAnySeries(series => {
    series.DataPadding = new LvcPoint(0, 0);
});
```

I also found the source code for the built-in light theme to be a useful reference when writing an extension method for a custom theme in my own application: https://github.com/beto-rodriguez/LiveCharts2/blob/master/src/skiasharp/LiveChartsCore.SkiaSharp/LightThemeExtensions.cs

**Reply by GF-Huang** [reply]
@mark-monteiro Thanks your example!

**Reply by GF-Huang** [reply]
@mark-monteiro Do you know the default value (width and height) of `DataPadding`? Because I only want to change width, and keep the height by default.

**Reply by beto-rodriguez** [reply]
@GF-Huang Every series uses a different value, I think the default padding for a line series in the Y axis is 1, so try `series.DataPadding = new LvcPoint(0, 1)`
